guides/source/error_reporting.md

```diff
@@ -33,72 +27,70 @@ rescue SomethingIsBroken => error
 end
 ```
 
-with
+with a consistent interface:
 
 ```rb
 Rails.error.handle(SomethingIsBroken) do
   do_something
 end
 ```
 
-This approach provides several benefits:
+Rails wraps all executions (such as HTTP requests, jobs, and `rails runner` invocations) in the error reporter, so any unhandled errors raised in your app will automatically be reported to your error-reporting service via their subscribers.
+
+This means that third-party error-reporting libraries no longer need to insert a Rack middleware or do any monkey-patching to capture unhandled exceptions. ActiveSupport libraries can also use this to non-intrusively report warnings that would previously have been lost in logs.
+
+Using the Rails' error reporter is not required. All other means of capturing errors still work.
 
-* Application or error reporting libraries don't need to insert a Rack middleware to capture unhandled exceptions from requests anymore.
-* To ActiveSupport-aware libraries, this can be used to report errors to the host application.
-* It decouples application code from error reporting libraries.
-* It reduces boilerplate code for handling exceptions.
-* Error reporting libraries will need less monkey-patches and be less intrusive to applications.
```

Comment:
Removed the bullet list of "benefits" as they made it look too academic, and didn't share the tone you normally get from the Rails docs. They were also not very distinct, and could be summarized in fewer sentences.